spec/models/claims/totals_spec.rb

```diff
@@ -236,45 +236,30 @@
     describe 'updating value bands and totals' do
       let(:claim) { create :litigator_claim }
 
-      it 'updates the value band id when an added disbursement takes it to the next band' do
-        expect(claim.total).to eq 25.0
-        expect(claim.vat_amount).to eq 0.0
-        expect(claim.value_band_id).to eq 10
+      context 'with a disbursment' do
+        let(:new_fee) { create :disbursement, claim:, net_amount: 30_000.0, vat_amount: 5_000.0 }
 
-        create :disbursement, claim: claim, net_amount: 25_000.0, vat_amount: 5_000.0
-
-        claim.reload
-        expect(claim.total).to eq 25_025.0
-        expect(claim.vat_amount).to eq 5_000.0
-        expect(claim.value_band_id).to eq 20
+        it { expect { new_fee }.to change(claim, :total).by(30_000) }
+        it { expect { new_fee }.to change(claim, :vat_amount).by(5_000) }
+        it { expect { new_fee }.to change { claim.reload.value_band_id }.from(10).to(20) }
```

Comment:
for clarification, the auth limit is affected by vat?

Reply:
Yes, the authority limit applies to the total value of the claim, including VAT. In CCCD `total` does not include VAT (it's the sum of all fees net of VAT) so we need to specify `vat_amount` separately.

Paul and I discussed this in a meeting with Nick Poulter and people from Finance and I think we agreed that this is the correct approach and consistent across all the apps. This PR is not changing the way it behaves, just the thresholds, so if it's working correctly now then it should continue to work correctly after the increase.

Reply:
ohh makes sense thank you. i'm getting myself confused XD